NiksVeinbergs/m2
# ML-1413 Ticket: Add Pre-Footer Text to CMS Page

### Description

The new function can be accessed through the Content->Pages->Edit Page->Pre-Footer Text fieldset. There should be an editable WYSIWYG form element.
Pre-Footer Text is saved in the "cms_page" table new field "pre_footer_text" (Added via SchemaPatch)
If Pre-Footer Text is added, it should be visible at the end of the CMS Page. (Before footer)
### Manual testing scenarios

1. Create new Pre-Footer Element in Content->Pages->Edit Page->Pre-Footer Text fieldset
2. Go to the page and check if the Pre-Footer block is visible at the end of the page.
3. Delete text in the Pre-Footer text fieldset
4. Check if text disappears on the CMS page
